resources/views/modulos.blade.php

```diff
@@ -7,6 +7,22 @@
                 <div class="card-header">
                     <h3>{{ __('Desarrollo de Aplicaciones Web') }}</h3>
                 </div>
+                <div class="row">
+
+                    @foreach( $arrayModulos as $modulo )
```

Comment:
Ya sabéis que, si os dan una vista, vuestra labor es programar la parte dinámica de la vista, pero sin modificar el aspecto.

En este caso, lo deberías haber mostrado en la tabla que se os facilitaba, utilizando el @foreach para generar tantas filas de la tabla (<tr></tr>) como módulos haya.